src/npm.js

```diff
@@ -20,6 +20,13 @@ const logWarning = ({ error, type, packagesStr }) => {
   );
 };
 
+export function validatePackageExists(pkgName) {
+  return got(`${c.npmRootEndpoint}/${pkgName}`, {
+    json: true,
+    method: 'HEAD',
+  }).then(response => response.statusCode === 200);
+}
```

Comment:
we only need to know if it exists - so you can use `curl -I https://registry.npmjs.org/@types/lodash` to see if it exists